src/Private/Pop-WindowsNotification.ps1

```diff
@@ -8,28 +8,43 @@ Function Pop-Notification {
 
         [Parameter(Mandatory=$true)]
         [string]
-        $Title
+        $Title,
+
+        [Parameter()]
+        [string]
+        $Icon = "$PSScriptRoot\lib\alarm.png",
+
+        [Parameter()]
+        [switch]
+        $Silent
     )
 
+    if ($Silent) {
+        $SoundElement = '<audio silent="true" />'
+    } else {
+        $SoundElement = '<audio src="ms-winsoundevent:Notification.Default" />'
+    }
+
     $XmlString = @"
     <toast>
     <visual>
         <binding template="ToastGeneric">
         <text>$Title</text>
         <text>$Body</text>
-        <image src="$PSScriptRoot\lib\alarm.png" placement="appLogoOverride" hint-crop="circle" />
+        <image src="$Icon" placement="appLogoOverride" hint-crop="circle" />
         </binding>
     </visual>
-    <audio src="ms-winsoundevent:Notification.Default" />
+    $SoundElement
     </toast>
 "@
 
-    $AppId = '{1AC14E77-02E7-4E5D-B744-2EB1AE5198B7}\WindowsPowerShell\v1.0\powershell.exe'
+    $AppId = Get-WindowsAppId
+
     $null = [Windows.UI.Notifications.ToastNotificationManager, Windows.UI.Notifications, ContentType = WindowsRuntime]
     $null = [Windows.Data.Xml.Dom.XmlDocument, Windows.Data.Xml.Dom.XmlDocument, ContentType = WindowsRuntime]
     $ToastXml = [Windows.Data.Xml.Dom.XmlDocument]::new()
     $ToastXml.LoadXml($XmlString)
+
     $Toast = [Windows.UI.Notifications.ToastNotification]::new($ToastXml)
     [Windows.UI.Notifications.ToastNotificationManager]::CreateToastNotifier($AppId).Show($Toast)
```

Comment:
Feel free to call this out of scope for this PR but we should add ShouldProcess here

Reply:
It can maybe print out the xml

Reply:
Have the ShouldProcess print the XML? I like it...

Reply:
Check out new commit!